nautobot/core/api/urls.py

```diff
@@ -56,6 +56,8 @@
     path("graphql/", GraphQLDRFAPIView.as_view(), name="graphql-api"),
     # Plugins
     path("plugins/", include((plugin_api_patterns, "plugins-api"))),
+    # Core (keeping for backwards compatibility for the moment)
```

Comment:
backwards compatibility with what? (i.e. please add more context to the comment :-))